README.md

```diff
@@ -51,6 +55,10 @@ Note: Hash should be in lowercase in index.json otherwise CI will fail.
 
 ### How to fill in the metadata for an index.json entry?
 
+The `azdev extension update-index` command can be used to simplify the process of updating the index file. Run `azdev extension update-index <URL>` where URL is the fully-qualified URL to your published extension WHL. This will gather the appropriate metadata and add an entry to your extension's metadata entries. For more information visit https://github.com/Azure/azure-cli-dev-tools.
```

Comment:
Instead of `add an entry to your extension's metadata entries.`, I recommend `add an entry to your extension in the index.`

Reply:
Reworded.